samples/bluetooth/bap_broadcast_sink/README.rst

```diff
@@ -1,8 +1,8 @@
 .. zephyr:code-sample:: bluetooth_bap_broadcast_sink
-   :name: Bluetooth: Broadcast Audio Sink
+   :name: Broadcast Audio Sink
    :relevant-api: bluetooth
 
-   Bluetooth: Broadcast Audio Sink
+   Use LE Audio Broadcast Sink functionality.
```

Comment:
I think I'll follow up with a name update PR later - I think several of our LE audio samples could benefit from better names and descriptions, but let's not muddy this PR too much with that :)

Reply:
Yes, this would be awesome. It's true that a lot of the names and descriptions are very "content-free" right now and not all that helpful. Hopefully the follow-up PR will be much easier to implement once the directives will already be there :)

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/78094 :)